.gitignore

```diff
@@ -35,3 +35,7 @@ Pods/
 # Carthage/Checkouts
 
 Carthage/Build
+
+Example/ChatExample.xcodeproj/project.pbxproj
```

Comment:
We shouldn't be adding this to .gitignore

Reply:
I am sorry. You are right. Let me try it again.